client/overview/index.js

```diff
@@ -144,27 +147,35 @@ const OverviewPage = () => {
 			{ ! accountRejected && (
 				<ErrorBoundary>
 					<>
-						<Card>
-							<Welcome />
-							<AccountBalances />
-						</Card>
+						{ showTaskList ? (
+							<>
+								<Card>
+									<Welcome />
+									<ErrorBoundary>
```

Comment:
I've wrapped the TaskList in an ErrorBoundary to allow the <Welcome /> component to render in the case of a TaskList error.